<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidget</class>
 <widget class="QWidget" name="DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>22</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>138</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_disp">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>体积：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_value">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(61, 61, 61);
background-color: rgb(122, 122, 122);</string>
       </property>
       <property name="text">
        <string>1.25</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>39</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>单位：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_unit">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(61, 61, 61);
background-color: rgb(122, 122, 122);</string>
       </property>
       <property name="text">
        <string>m³</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
